modules/validations/validation_issue.js

```diff
@@ -0,0 +1,41 @@
+import _isObject from 'lodash-es/isObject';
+
+
+var ValidationIssueType = Object.freeze({
+    deprecated_tags: 'deprecated_tags',
+    disconnected_highway: 'disconnected_highway',
+    many_deletions: 'many_deletions',
+    missing_tag: 'missing_tag',
+    old_multipolygon: 'old_multipolygon',
+    tag_suggests_area: 'tag_suggests_area',
+    map_rule_issue: 'map_rule_issue',
+});
+
+
+var ValidationIssueSeverity = Object.freeze({
+    warning: 'warning',
+    error: 'error',
+});
+
+
+export { ValidationIssueType, ValidationIssueSeverity };
+
+
+export function validationIssue(attrs) {
```

Comment:
can this be `ValidationIssue`?

Reply:
I was trying to follow the convention of other "classes" like geoExtent, osmEntity, etc.